services/lsst-dev.rst

```diff
@@ -99,7 +99,7 @@ You can also following in the `~/.bashrc` file (this is the jist of the :file:`l
 
 .. code-block:: bash
 
-   source ~lsstsw/eups/bin/setups.sh   # bash users
+   source ~lsstsw/eups/current/bin/setups.sh   # bash users
    setup anaconda
```

Comment:
Actually these two setups worry me a bit. The `git` is not needed because of `devtoolset-3` and the `anaconda` may well lead to an old version of python being set up. You need the python that was used to build the `lsstsw` stack and that is in `~lsstsw/miniconda/bin`.